gnomad/utils/constraint.py

```diff
@@ -360,3 +412,279 @@ def collapse_strand(
+    mutation rate estimates against the proportion observed of each substitution,
+    context, and methylation level in `coverage_ht`.
+
+    Two plateau models are fit, one for CpG transitions, and one for the remainder of
+    sites (transversions and non CpG transitions).
+
+    The plateau models only consider high coverage sites, or sites above a median
+    coverage of `cov_cutoff`.
+
+    Plateau model: adjusts proportion of expected variation based on location in the
+    genome and CpG status.
+    The x and y of the plateau models:
+    - x: `mu_snp` - mutation rate
+    - y: proportion observed ('observed_variants' or 'observed_{pop}' / 'possible_variants')
+
+    This function also builds models (coverage models) to calibrate the proportion of
+    expected variation at low coverage sites (sites below `cov_cutoff`).
+
+    The coverage models are built by creating a scaling factor across all high coverage
+    sites, applying this ratio to the low coverage sites, and running a linear
+    regression.
+
+    Coverage model: corrects proportion of expected variation at low coverage sites.
+    Low coverage sites are defined as sites with median coverage < `cov_cutoff`.
+
+    The x and y of the coverage model:
+    - x: log10 groupings of exome coverage at low coverage sites
+    - y: sum('observed_variants')/ (`high_coverage_scale_factor` * sum('possible_variants' * 'mu_snp') at low coverage sites
+
+    `high_coverage_scale_factor` = sum('observed_variants') /
+                        sum('possible_variants' * 'mu_snp') at high coverage sites
+
+
+    .. note::
+
+        This function expects that the input Table(`coverage_ht`) was created using
+        `get_proportion_observed_by_coverage`, which means that `coverage_ht` should
+        contain only high quality synonymous variants below 0.1% frequency.
+
+        This function also expects that the following fields are present in
+        `coverage_ht`:
+        - context - trinucleotide genomic context
+        - ref - the reference allele
+        - alt - the alternate base
+        - methylation_level - methylation level
+        - cpg - whether the site is CpG site
+        - exome_coverage - median exome coverage at integer values between 1-100
+        - observed_variants - the number of observed variants in the dataset for each
+        variant. Note that the term "variant" here refers to a specific substitution,
+        context, methylation level, and coverage combination.
+        - downsampling_counts_{pop} (optional) - array of observed variant counts per
+        population after downsampling. Used only when `pops` is specified.
+        - mu_snp - mutation rate
+        - possible_variants - the number of possible variants in the dataset for each
+        variant.
+
+    :param coverage_ht: Input coverage Table.
+    :param weighted: Whether to weight the high coverage model (a linear regression
```

Comment:
```suggestion
    :param weighted: Whether to weight the plateau models (a linear regression
```
this doesn't actually weight the coverage models at the moment